pkg/controller/attach_detach.go

```diff
@@ -127,20 +127,36 @@ func (r *ReconcileAttachDetach) Reconcile(ctx context.Context, request reconcile
 }
 
 func (r *ReconcileAttachDetach) triggerAttach(ctx context.Context, azVolumeAttachment *diskv1beta1.AzVolumeAttachment) error {
+	var err error
+	var waitForGoroutine bool
+	var aborted bool
+	ctx, workflow := azureutils.NewWorkflow(ctx)
```

Comment:
Should we pass the AzVolumeAttachment name/id as details while creating NewWorkflow(), so that we could also search  the logs by that Id?

Reply:
Given that the logger details contain the volume name and node name, wouldn't it be sufficient to query with the volume `AND` node keyword?